@ndlib/gatsby-theme-marble/src/components/Shared/ActionButtonGroup/DownloadButton/DownloadModalContent/DownloadCitation/index.js

```diff
@@ -1,5 +1,12 @@
-const DownloadCitation = ({ iiifManifest }) => {
-  return null
+import React from 'react'
```

Comment:
Since we're going to use `theme-ui`, instead of importing react change the first two lines to the following:
```
/** @jsx jsx */
import { jsx, Styled } from 'theme-ui'
```